internal/controllers/utils/utils.go

```diff
@@ -574,25 +603,34 @@ func GetConfigmap(ctx context.Context, c client.Client, name, namespace string)
 	if !cmExists {
 		// Check if the configmap is missing
 		return nil, NewInputError(
-			"the ConfigMap %s is not found in the namespace %s", name, namespace)
+			"the ConfigMap '%s' is not found in the namespace '%s'", name, namespace)
 	}
 	return existingConfigmap, nil
 }
 
+// GetConfigMapField attempts to retrieve the value of the field using the provided field name
+func GetConfigMapField(cm *corev1.ConfigMap, fieldName string) (string, error) {
+	data, ok := cm.Data[fieldName]
+	if !ok {
+		return data, NewInputError("the ConfigMap '%s' does not contain a field named '%s'", cm.Name, fieldName)
+	}
+
+	return data, nil
+}
+
 // ExtractTemplateDataFromConfigMap extractes the template data associated with the specified key
```

Comment:
I know, this was here before, but typo: `extracts`

Reply:
fixed